pkg/miniogw/gateway-storj.go

```diff
@@ -100,34 +102,40 @@ func (s *storjObjects) GetBucketInfo(ctx context.Context, bucket string) (
 	return minio.BucketInfo{Name: bucket, Created: meta.Created}, nil
 }
 
-func (s *storjObjects) GetObject(ctx context.Context, bucket, object string,
-	startOffset int64, length int64, writer io.Writer, etag string) (err error) {
+func (s *storjObjects) getObject(ctx context.Context, bucket, object string) (rr ranger.RangeCloser, err error) {
 	defer mon.Task()(&ctx)(&err)
 	o, err := s.storj.bs.GetObjectStore(ctx, bucket)
 	if err != nil {
-		return err
+		return nil, err
 	}
-	rr, _, err := o.Get(ctx, paths.New(object))
+
+	rr, _, err = o.Get(ctx, paths.New(object))
+	if err != nil {
```

Comment:
`return rr, err` would be equivalent to the this and the next 4 lines, but would be cleaner code.